src/config.ts

```diff
@@ -137,6 +142,22 @@ export interface Config {
    */
   samplingRate?: number;
 
+  /**
+   * Specifies how to use incoming trace context headers. The following options
+   * are available:
+   *   'default' -- Trace context headers will be respected for incoming
+   *   requests that support it. A new trace will be created for requests
+   *   without trace context headers.
+   *   'require' -- Same as default, but traces won't be created for requests
+   *   without trace context headers. This should not be set for end user-facing
+   *   services, as this header is usually set by other traced services rather
+   *   than by users.
+   *   'ignore' -- Always ignore trace context headers, so a new trace with a
+   *   unique ID will be created for every request.
```

Comment:
The original comment for `ignoreContextHeader` did a much better job explaining when this is useful. As part of your transition you have lost that comment.

Reply:
I've updated it to try to include the accurate portion of the docs (`Settings this to true... same incoming request.` is misleading).